### PR TITLE
feat: add further logging to absvc toggle resolution logic

### PR DESCRIPTION
**Pull request by cameronmeissner**
**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:
- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

File: pkg/agent/toggles/types.go [hunk 1]
```diff
@@ -72,19 +77,32 @@ func (t *Toggles) getMap(name string, entity *Entity) map[string]string {
 		log.Printf("map toggles are nil, resolving to default empty map value for toggle: %q", name)
 		return map[string]string{}
 	}
+	value := map[string]string{}
 	if toggle, ok := t.Maps[name]; ok {
-		return toggle(entity)
+		value = toggle(entity)
 	}
-	return map[string]string{}
+	log.Printf("resolved value of toggle %q; entity: %s; value: %s", name, entity, marshalToString(value))
```
Comment (lilypan26) [inline pkg/agent/toggles/types.go:84]: Should `entity` be converted to string here? `entity.String()`?

File: pkg/agent/toggles/types.go [hunk 2]
```diff
@@ -72,19 +77,32 @@ func (t *Toggles) getMap(name string, entity *Entity) map[string]string {
...
 func (t *Toggles) getString(name string, entity *Entity) string {
 	if t == nil || t.Strings == nil {
 		log.Printf("string toggles are nil, resolving to default empty string value for toggle: %q", name)
 	}
+	var value string
 	if toggle, ok := t.Strings[name]; ok {
-		return toggle(entity)
+		value = toggle(entity)
+	}
+	log.Printf("resolved value of toggle %q; entity: %s; value: %s", name, entity, value)
```
Comment (lilypan26) [inline pkg/agent/toggles/types.go:97]: same a above.

Comment (cameronmeissner) [reply pkg/agent/toggles/types.go:97]: since it implements Stringer go will automatically call `String()` :)

similar to java and `toString()`

Comment (lilypan26) [reply pkg/agent/toggles/types.go:97]: Ah cool! thanks for clarifying

**Review by lilypan26 — APPROVED** [review]

**Review by anujmaheshwari1 — APPROVED** [review]

**Pull request merged**